elements/noflo-new-project.html

```diff
@@ -72,6 +74,7 @@
         this.fire('new', {
           id: this.projectId.replace(/[\/\s\.]/g, '-'),
           name: this.name,
+          namespace: this.namespace.replace(/[\/\s\.]/g, '-'),
```

Comment:
Shouldn't this already be fine from the replacements in `nameChanged`? And why are the replacement logics there different?